symfony/client/src/orangehrmAuthenticationPlugin/pages/reset-password.scss

```diff
@@ -0,0 +1,93 @@
+/*!
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+@import '@ohrm/oxd/styles/_colors.scss';
+@import '@ohrm/oxd/styles/_mixins.scss';
+
+$card-padding-left-right: 1.2rem;
+$card-padding-top-bottom: 0.6rem;
+$card-padding: $card-padding-top-bottom $card-padding-left-right;
+
+.orangehrm-card-container {
+  box-shadow: 3px 3px 10px $oxd-interface-gray-color;
+  width: 80vw;
+  @media only screen and (min-width: 800px) {
+    width: 30vw;
+  }
```

Comment:
@super-chama

Reply:
@Ajiharan007  from `@ohrm/oxd/styles/_mixins.scss` you can use `oxd-respond-to` mixin

```
.orangehrm-card-container {
  width: 30vw;
  box-shadow: 3px 3px 10px $oxd-interface-gray-color;
  @include oxd-respond-to('md') {
     width: 80vw;
  }
}
```